middleware/util/logging/instance.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"fmt"
+	"path/filepath"
+	"sync"
+
+	"github.com/digitalbitbox/bitbox-wallet-app/util/config"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/errp"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// configFileName stores the name of the file that contains the logging configuration.
+	configFileName = "logging.json"
+)
+
+var instance *Logger
+var once sync.Once
+
+// Get returns the configured logger or a new one based on the configuration file.
+func Get() *Logger {
+	once.Do(func() {
```

Comment:
I think it might be more readable to define a `var` that holds the anonymous `func() {...}`, either inline in `Get()` or as a top-level helper function.